extension/observer/cfgardenobserver/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package cfgardenobserver // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/cfgardenobserver"
+
+import (
+	"errors" // Config defines configuration for cf garden observer.
```

Comment:
I suspect that comment is a side affect of formatter tripping up.

Do you mind fixing up the placement?